core/src/components/range/range.ionic.scss

```
@@ -107,12 +107,11 @@
 }
 
 .range-pin {
-  @include transform(translate3d(0, 100%, 0), scale(0.01));
-  @include padding(globals.$ion-space-200, globals.$ion-space-200, globals.$ion-space-200, globals.$ion-space-200);
+  @include padding(globals.$ion-space-300, globals.$ion-space-300, globals.$ion-space-300, globals.$ion-space-300);
 
   min-width: 28px;
 
-  transition: transform 120ms ease;
+  transform: translate3d(0, calc(-100% + 11px), 0) scale(1);
```

```suggestion
  transform: translate3d(0, calc(-100%), 0);
```

We can transform by the pin height by itself since the padding on it handles the space between the knob and text